cpp/src/gandiva/gdv_function_stubs_test.cc

```diff
@@ -766,4 +766,62 @@ TEST(TestGdvFnStubs, TestCastVarbinaryFloat8) {
   ctx.Reset();
 }
 
+TEST(TestGdvFnStubs, TestInstr) {
+  std::string s1 = "hello world!";
+  auto s1_len = static_cast<int32_t>(s1.size());
+  std::string s2 = "world";
+  auto s2_len = static_cast<int32_t>(s2.size());
+
+  auto result = gdv_fn_instr_utf8(s1.c_str(), s1_len, s2.c_str(), s2_len);
+  EXPECT_EQ(result, 7);
+
+  s1 = "apple banana mango";
```

Comment:
Please add a test where string = "Hello" and substring = "Hello". Another where string = "Hello", substring = "Hell"

Reply:
Added